cmd/kube-proxy/app/server_others.go

```diff
@@ -193,6 +195,14 @@ func newProxyServer(
 		if err != nil {
 			return nil, fmt.Errorf("unable to create proxier: %v", err)
 		}
+		if utilfeature.DefaultFeatureGate.Enabled(features.IPv6DualStack) {
+			klog.V(0).Info("IPv6DualStack; Creating MetaProxier for ipvs.")
+			ipv6Proxier, err := ipvs.NewOtherProxyServer(config)
```

Comment:
Why do we need `other`, they are the same proxy working on a different family.

Reply:
For dual-stack two instances of the proxier are needed (and a "meta-proxier" on top). So compared to single-stack a new proxier instance must be created for the "other" (not-cluster) family. The proxier examines the family of `nodeIP` to determine if it shall work in ipv4 or ipv6 mode. 

The `ipset` names are modified internally already. The `isIPv6` parameter to `NewIPSet` determines the operation. But there is a race problem with ipset and ipvsadm pointed out by @andrewsykim that I will get back to later.

The reason for the `NewOtherProxyServer` function is that I wanted to keep the new code in the "ipvs" package. The proxier instance for the *non-cluster* (other) family can be created by duplicating the code for creating the first proxier in `server_others.go`. Perhaps that is better for maintenace even if it means more code in a common package?

Reply:
After reading https://github.com/kubernetes/kubernetes/pull/80627#pullrequestreview-277510712 I think get your point.